ansible/roles/set-repositories/tasks/satellite-repos.yml

```diff
@@ -19,27 +19,16 @@
     name: katello-ca-consumer-*.noarch
     state: absent
 
-- name: Download Cert from Satellite
-  get_url:
-    url: "https://{{ set_repositories_satellite_url | default(satellite_url) }}/pub/katello-ca-consumer-latest.noarch.rpm"
-    dest: /root/katello-ca-consumer-latest.noarch.rpm
-    mode: 0664
-    validate_certs: no
-  register: r_get_cert
-  until: not r_get_cert.failed
+- name: Install cert
+  tags: packer
+  package:
+    name: "https://{{ set_repositories_satellite_url | default(satellite_url) }}/pub/katello-ca-consumer-latest.noarch.rpm"
+    state: present
+  register: r_install_cert
+  until: not r_install_cert.failed
```

Comment:
When run from a server that does not have `/etc/pki/ca-trust/source/anchors/katello-server-ca.pem` this task fails with the error:
```
Failure downloading https://labsat.opentlc.com/pub/katello-ca-consumer-latest.noarch.rpm, Request failed: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:897)>
```

The ca-trust anchor is provided by this package, so it seems the only way this works is if it is run on a host that has already previously installed this package.

The ca-trust file will not show as owned by this package because it is installed during post scripts:

```
# rpm -qf /etc/pki/ca-trust/source/anchors/katello-server-ca.pem
error: file /etc/pki/ca-trust/source/anchors/katello-server-ca.pem: No such file or directory
# rpm -qpl katello-ca-consumer-latest.noarch.rpm
/usr/bin/katello-rhsm-consumer
# rpm -qp --scripts katello-ca-consumer-latest.noarch.rpm
postinstall scriptlet (using /bin/sh):
/bin/bash /usr/bin/katello-rhsm-consumer
postuninstall scriptlet (using /bin/sh):
test -f /etc/rhsm/rhsm.conf.kat-backup && command cp /etc/rhsm/rhsm.conf.kat-backup /etc/rhsm/rhsm.conf
```